app/components/filters/provider_attributes.rb

```diff
@@ -0,0 +1,12 @@
+# frozen_string_literal: true
+
+module Filters
+  class ProviderAttributes < ViewComponent::Base
+    attr_accessor :filters
+
+    def initialize(filters:)
+      @filters = filters
+      super
```

Comment:
```suggestion
      super
      @filters = filters
```

Always call `super` **before** initialising instance variables